client/src/pages/Guess.jsx

```diff
@@ -104,21 +122,32 @@ const Guess = () => {
                     </div>
 
                     {selectedPanel === 'map' &&
-                        <GoogleMap
-                            center={center}
-                            zoom={2.5}
-                            mapContainerClassName='guess-panel-container'
-                            options={{
-                                streetViewControl: false,
-                                fullscreenControl: false,
-                                gestureHandling: 'greedy',
-                                minZoom: 2.5,
-                            }}
-                            onLoad={map => setMap(map)}
-                            onClick={onClickMap}
-                        >
-                            <Marker key={markerKey} position={markerKey ? markerPosition : { lat: null, lng: null }} />
-                        </GoogleMap>
+                        <div>
+                            <div className="home-post">
+                                <div className="rectangle">
+                                    <div className="image">
+                                        <GoogleMap
+                                            center={center}
```

Comment:
No action needed for now. But in the future I wonder if we can move this GoogleMap element to its separate components file